src/main/java/qna/domain/Question.java

```diff
@@ -30,24 +36,25 @@ public Question(Long id, String title, String contents) {
     }
 
     public Question writeBy(User writer) {
-        this.writerId = writer.getId();
+        this.writer = writer;
         return this;
     }
 
     public boolean isOwner(User writer) {
-        return this.writerId.equals(writer.getId());
+        return this.writer.equals(writer);
     }
 
     public void addAnswer(Answer answer) {
+        answers.add(answer);
```

Comment:
```suggestion
        if (!answers.contains(answer)) {
            answers.add(answer);
        }
```
이렇게 validation 체크를 한번 해줘도 좋을 것 같습니다 ㅎㅎ